test/math.jl

```diff
@@ -362,3 +362,22 @@ end
 for z in (1.234, 1.234 + 5.678im, [1.234, 5.678])
     @test_approx_eq cis(z) exp(im*z)
 end
+
+# modf
+for elty in (Float32, Float64)
+    @test_approx_eq modf( convert(elty,1.2) )[1] convert(elty,0.2)
```

Comment:
I guess one slightly more concise syntax for testing approximate equality of a tuple-returning function would be by splatting into an array like `@test_approx_eq [modf(convert(elty,1.2))...] elty[0.2, 1.0]`. There's more syntax features going on there so maybe that's unnecessarily complicated.